<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>401</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>File loading</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Block size: (in kilobytes)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="blockSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Warning: Changing the block size will reload any opened file from disk!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>File viewing</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrap mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="wrapModeComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Wrap</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Word Wrap</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wrap Anywhere</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wrap At Word Boundary Or Anywhere</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>File editing</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Write mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="writeModeComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Simple</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Complex</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
